logical/framework/backend.go

```diff
@@ -511,6 +511,12 @@ type FieldSchema struct {
 	// output as part of OpenAPI generation and may effect documentation and
 	// dynamic UI generation.
 	AllowedValues []interface{}
+
+	// Display* members are available to provide hints for UI and documentation
+	// generators. They will be included in OpenAPI output if set.
+	DisplayName      string
+	DisplayValue     interface{}
+	DisplaySensitive bool
```

Comment:
Could you add a comment explaining what the `DisplayValue` field is, and maybe even going so far as to add that it should not be viewed as or used as a default in case other developers look it up in the future?

Also, would you mind likewise commenting `DisplaySensitive`? I'm quite curious about this field and what its effects would be if I toggled it.

Reply:
Yep. Added some more comments. DisplayValue is a bit tough to describe, as evidenced by our eco meeting discussion this morning.